### ISSUE TITLE
 Error in load_data_detection() - OpenCV 

### ISSUE THREAD

**Issue by Lerseb** [AlexeyAB/darknet#6779]
While training, it runs for 1 or 2 minutes and then I get this message.
I am not sure about the path to use in the train.txt. Is is the full path (home/...data/obj/name of the image) or just the path from the folder where I opened the terminal window (data/obj/name of the image).
Or maybe it is an issue with my version of opencv which is not able to open the file. 
Below the last bloc of lines from my terminal :
Thanks for helping.

 total_bbox = 15027, rewritten_bbox = 0.019964 % 
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 139 Avg (IOU: 0.000000, GIOU: 0.000000), Class: 0.000000, Obj: 0.000000, No Obj: 0.497379, .5R: 0.000000, .75R: 0.000000, count: 1, class_loss = 2102.348877, iou_loss = 0.000000, total_loss = 2102.348877 
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 150 Avg (IOU: 0.124697, GIOU: -0.268941), Class: 0.772755, Obj: 0.498636, No Obj: 0.545832, .5R: 0.000000, .75R: 0.000000, count: 3, class_loss = 643.884644, iou_loss = 0.029480, total_loss = 643.914124 
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 161 Avg (IOU: 0.240064, GIOU: -0.092025), Class: 0.718445, Obj: 0.244284, No Obj: 0.536518, .5R: 0.000000, .75R: 0.000000, count: 1, class_loss = 157.912979, iou_loss = 0.009430, total_loss = 157.922409 
 total_bbox = 15031, rewritten_bbox = 0.019959 % 

 Tensor Cores are disabled until the first 3000 iterations are reached.

 25: 974.492859, 976.366943 avg loss, 0.000000 rate, 3.015330 seconds, 1600 images, 15.088839 hours left
Loaded: 0.000035 seconds
Cannot load image ﻿data/obj/IMG_1833(4,7)c0e7a2ed-2305-4e35-ae37-9214a0b2a7b8.jpg

 Error in load_data_detection() - OpenCV 
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 139 Avg (IOU: 0.000000, GIOU: 0.000000), Class: 0.000000, Obj: 0.000000, No Obj: 0.496547, .5R: 0.000000, .75R: 0.000000, count: 1, class_loss = 2103.472656, iou_loss = 0.000000, total_loss = 2103.472656 
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 150 Avg (IOU: 0.000000, GIOU: 0.000000), Class: 0.000000, Obj: 0.000000, No Obj: 0.550499, .5R: 0.000000, .75R: 0.000000, count: 1, class_loss = 645.998169, iou_loss = 0.000000, total_loss = 645.998169 
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 161 Avg (IOU: 0.000000, GIOU: 0.000000), Class: 0.000000, Obj: 0.000000, No Obj: 0.537672, .5R: 0.000000, .75R: 0.000000, count: 1, class_loss = 156.021637, iou_loss = 0.000000, total_loss = 156.021637 
 total_bbox = 15031, rewritten_bbox = 0.019959 % 
Segmentation fault (core dumped)

**Comment by Lerseb** [comment]
I figured out what to do to avoid Segmentation fault (core dumped)
1- Check txt files for empty ones => delete them
2- Check txt files for 0, 0 coordinates => Remove object with such coordinates
3- Don't trust automatically prepared list of jpeg files for train.txt and test.txt
(ie run command ls in terminal where the jpeg files are. Then copy all files to a text editor and use find and replace to make necessary changes on the path)
Best is to split your dataset like 20% test and 80% train.
Hope this can help.😉
